modules/installation-ibm-z-user-infra-machines-iso.adoc

```diff
@@ -62,6 +64,40 @@ rd.znet=qeth,0.0.bdf0,0.0.bdf1,0.0.bdf2,layer2=1,portno=0 zfcp.allow_lun_scan=0
 !condev rd.dasd=0.0.3490
 ----
 
+
+** For installations on FCP type disks, specify:
+... `rd.zfcp=<adapter>,<wwpn>,<lun>` specifies the FCP disk where {op-system} is to be installed. For multipathing repeat this step for each additional path. 
+... For multipathing the following parameter must be set: `rd.multipath=default`
+... For multipathing the install device is: `coreos.inst.install_dev=/dev/mapper/mpatha`
+... For single path the install device is: `coreos.inst.install_dev=sda`
++
+[NOTE]
+====
+If additional LUNs are configured with NPIV, FCP requires `zfcp.allow_lun_scan=0`.  
+====
+... All other parameters can stay as they are.
++
+[NOTE]
+====
+Additional post-installation steps are required to fully enable multipathing. See “Completing installation on user-provisioned infrastructure” for more information.  
+====
++
+Example parameter file, `bootstrap-0.parm`, for the bootstrap machine with multipathing:
```

Comment:
The following is an example parameter file `bootstrap-0.parm` for the bootstrap machine with multipathing: